sdks/java/core/src/main/java/org/apache/beam/sdk/values/Row.java

```diff
@@ -355,6 +379,11 @@ public Builder addValues(Object ... values) {
       return this;
     }
 
+    public <T1, T2> Builder addMap(Map<T1, T2> data) {
```

Comment:
<!--new_thread; commit:7bf9d2048e8d568ee9280629db391775cc490358; resolved:0-->
Is this overload needed? addArray was needed because otherwise passing an array into addValues tended to unroll the array, but I don't think Java will do that for maps.

Reply:
it's not necessary, will remove.